test/transforms/test_node_property_split.py

```diff
@@ -0,0 +1,64 @@
+from functools import reduce
+from itertools import combinations
+
+import numpy as np
+import torch
+
+from torch_geometric.datasets import graph_generator
+from torch_geometric.transforms import NodePropertySplit
+
+
+def test_node_property_split():
```

Comment:
You might need to add a decorator `@withPackage('networkx')` here to add the `networkx` package during tests.

Reply:
It works, thank you!